### PR TITLE
Show appropriate terraform logo for light and dark themes in README

### PR DESCRIPTION
**Pull request by thekbb**
The README looks janky on GitHub when using the dark theme.
Cursor_and_okta_terraform-provider-okta__Terraform_Okta_provider

This will swap out [svg's linked from Hashicorp's terraform brand page ](https://www.hashicorp.com/brand)based on the users GitHub theme
default light (for monsters that use this):
default-light
default dark:
default-dark

The `<picture>` element is well understood, [GitHub's docs indicate this is the preferred way to get this done](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to).

### CODE REVIEW

**Comment by virgofx** [comment]
@thekbb Would it be possible to include the assets locally in this repo instead of a random 3rd party? Then reference them similarly to the old asset.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 <br/><br/>
 
 <a href="https://terraform.io">
-    <img src="https://raw.githubusercontent.com/hashicorp/terraform-website/master/public/img/logo-text.svg" alt="Terraform logo" title="Terraform" height="50" />
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="https://www.datocms-assets.com/2885/1620155114-brandhcterraformprimaryattributedcolorwhite.svg">
+        <source media="(prefers-color-scheme: light)" srcset="https://www.datocms-assets.com/2885/1620155113-brandhcterraformprimaryattributedcolor.svg">
```
Comment (thekbb) [inline README.md]: These links are from Terraform's brand page... Now that I think on it a bit, I should upload them in a comment which will generate a https://user-images.githubusercontent.com URL for each, which will insulate this repo from changes to https://www.hashicorp.com/brand

**Comment by thekbb** [comment]
> @thekbb Would it be possible to include the assets locally in this repo instead of a random 3rd party? Then reference them similarly to the old asset.

jinx. I was thinking a similar thing in https://github.com/okta/terraform-provider-okta/pull/1574/files#r1207362748

I'll go with your plan, which is better.

**Comment by virgofx** [comment]
Ah, didn't know these were from Terraform's brand page. I guess it would be okay to leave. My first thought when I saw the domain was like -- what is that? what happens if that 3rd party hosting goes away kind of thing.

**Comment by thekbb** [comment]
> Ah, didn't know these were from Terraform's brand page. I guess it would be okay to leave. My first thought when I saw the domain was like -- what is that? what happens if that 3rd party hosting goes away kind of thing.

It's still probable that someone at Hashicorp reorganizes their stuff, they'd never imagine people would link to that. 

I added it to the repo, and think it's much better. Thank you for the suggestion, @virgofx.

**Review by monde — APPROVED** [review]
Awesome effort @thekbb , thank you so much!

**Pull request merged**